static/home/css/alignment.css

```diff
@@ -116,6 +116,12 @@
     color: #000000;
     text-decoration: none;
 }
+.ali-scroll-div {
+  position: -webkit-sticky;
```

Comment:
Does this work across browsers? Or is it meant to prevent the issue from happening in Safari and Chrome?
I've seen issues with webkit when one is using Brave, for example.

Reply:
As discussed 😄 (+ tested on Brave, seemed to be happy)